content/en/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md

```diff
@@ -7,6 +7,12 @@ slug: sig-usability-spotlight-2021
 
 **Author:** Kunal Kushwaha, Civo
 
+_**Note: SIG Usability is Not active.**_
+_SIG Usability, which is featured in this Spotlight blog, has been deprecated and is no longer active.
+As a result, the links and information provided in this post may no longer be valid or relevant.
+Should there be renewed interest and increased participation in the future, the SIG may be revived. However, for the time being, it will be inactive as per the k8s/community policy.
```

Comment:
```suggestion
Should there be renewed interest and increased participation in the future, the SIG may be revived. However, for the time being, it will be inactive as per the Kubernetes community policy.
```